pkg/utils/api/image.go

```diff
@@ -90,6 +92,28 @@ func extract(obj interface{}, path []string, keyPath, valuePath string, fields [
 		if !ok {
 			return fmt.Errorf("invalid value")
 		}
+
+		if jmesPath != "" {
+			jmesPath = strings.ReplaceAll(jmesPath, "$image", fmt.Sprintf(`'%s'`, value))
```

Comment:
why are we doing this ? instead of passing the value to the jmespath filter ?

Reply:
Now passing value to jmespath filter